L1Trigger/CSCTriggerPrimitives/src/CSCComparatorDigiFitter.cc

```diff
@@ -0,0 +1,312 @@
+#include "L1Trigger/CSCTriggerPrimitives/src/CSCComparatorDigiFitter.h"
+#include "L1Trigger/CSCCommonTrigger/interface/CSCConstants.h"
+#include "DataFormats/Math/interface/normalizedPhi.h"
+
+namespace
+{
+  // CSC LCT patterns
+  // the number quotes the distance to the center
+
+  // pid=0: no pattern found
+  std::vector<std::vector<int> > pat0delta;
```

Comment:
Hi @dildick - thanks - but I think your code requires this to be pat0delta(6) to fill the outer vector with empty vectors